Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.kt

```diff
@@ -1265,7 +1265,7 @@ class SettingsFragmentPresenter(
             )
         )
 
-        if (GpuDriverHelper.supportsCustomDriverLoading() && this.gpuDriver != null) {
+        if (GpuDriverHelper.supportsCustomDriverLoading() && this.gpuDriver != null && this.gameId.isNullOrEmpty()) {
```

Comment:
```suggestion
        if (GpuDriverHelper.supportsCustomDriverLoading() && this.gpuDriver != null && this.gameId.isNullOrEmpty() && !NativeLibrary.IsRunning()) {
```
Do this and remove the item you put in the StringSetting array